.gitignore

```diff
@@ -128,4 +128,5 @@ dmypy.json
 # Pyre type checker
 .pyre/
 
+# PyCharm project files
 .idea
```

Comment:
It's ok, but can we also add newline at the EOF?

Reply:
Fixed in 76d43a6